### DISCUSSION TITLE
开机启动后，为何无法自动在后台运行？

### DISCUSSION THREAD

**Discussion by star2k** [localsend/localsend#161] [L-简体中文]
开机启动后，为何无法自动在后台运行，每次都会在桌面显示ui？后台相关选项无效

**Comment by 601958483** [comment]
你可以手动将软件的快捷方式复制一份到这个路径下，就会自启动，但是好像不能“静默自启:只启动托盘服务”

**Reply by Tienisto** [reply]
Add the `autostart` parameter: #71

**Comment by 601958483** [comment]
C:\Users\60195\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup
